### PR TITLE
[CollectionView] UWP- Updating the ItemsLayout type should refresh the layout

### PR DESCRIPTION
**Pull request by EmilAlipiev**
This is an update to this [issue](https://github.com/xamarin/Xamarin.Forms/issues/5354). It was closed with fix for Android and IOS but UWP wasnt  working. 
I have tried my best to fix the issue. I am not sure if that is the best way of fixing this. But it works fine. I didnt create a new test case because existing case 5354 can be used and it works fine like that.

### CODE REVIEW

**Comment by EmilAlipiev** [comment]
can we get this fix? if anyone can review and accept it? it is so important for uwp :)

**Comment by EmilAlipiev** [comment]
Can we get this fix merged please? This bug makes collectionview not usable for uwp.

File: Xamarin.Forms.Platform.UAP/CollectionView/StructuredItemsViewRenderer.cs [hunk 1]
```diff
@@ -186,6 +186,25 @@ protected override void HandleLayoutPropertyChanged(PropertyChangedEventArgs pro
 						break;
 				}
 			}
+			else if (property.Is(StructuredItemsView.ItemsLayoutProperty))
```
Comment (jsuarezruiz) [inline Xamarin.Forms.Platform.UAP/CollectionView/StructuredItemsViewRenderer.cs:189]: Changing the ItemsLayout used, switch between `FormsListView` and `FormsGridView` with the appropriate container in `ItemsViewRenderer.cs`, this block is not necessary.

**Comment by jsuarezruiz** [comment]
To fix https://github.com/xamarin/Xamarin.Forms/issues/10482 I created this related PR https://github.com/xamarin/Xamarin.Forms/pull/10797. The issue is the same. So, take a look to this PR. After updating this PR we can link the issue and close the other PR. Let me know if you need help with something.

**Comment by EmilAlipiev** [comment]
Hi @jsuarezruiz thanks for your reply. I have just checked your PR. it looks a bit different than mine. If you think that your one resolves the issue better i dont mind, please go ahead. I was just trying my best since nobody picked up this issue almost a year :) For me the most important thing is that we can use collectionview on uwp ;)

**Comment by jsuarezruiz** [comment]
@EmilAlipiev I understand it and I really appreciate the effort, thanks!. We want the same, allow you use CollectionView in UWP without problems. If you complete the PR, I will be happy to review/help with it, etc.

**Comment by EmilAlipiev** [comment]
@jsuarezruiz ok i applied your changes and pushed. please go ahead :) i hope to see this fix merged soon.

**Comment by jsuarezruiz** [comment]
Thanks for the changes. I have done tests and the behavior is now correct.

82464837-d1a5f700-9abe-11ea-9e19-7a4f96b3836e

@samhouts should we change the target to 4.7?. On the other hand, we could close #10797 or merge it too since it adds a specific sample to Core Gallery, etc.

**Comment by EmilAlipiev** [comment]
@jsuarezruiz  what is that conflict now?  Do you have any idea how we can resolve it?

**Comment by rmarinho** [comment]
Hi @EmilAlipiev i fixed the conflict but seems the branch is not correct, there are changes to files that are not needed. Like FormsProgressBar

**Comment by EmilAlipiev** [comment]
Hi @rmarinho yes, i made a mistake yesterday and worked on another issue without creating a new branch. i have recognized after i pushed them. i need to separate them to a new branch. It is the confusion of working in a fork branch :)

**Comment by rmarinho** [comment]
@EmilAlipiev please onen a new pr when ready. 

Thanks

**Comment by EmilAlipiev** [comment]
@rmarinho Should I open another PR for ProgressRing changes do you mean? I am bit confused here because when I look at the origin/master history, it shows my commit was merged.  Is there something I am missing here? 

image

**Comment by rmarinho** [comment]
@EmilAlipiev this PR shows changes to the ProgressRing , but the title refers to ItemsLayout and CollectionView.

**Pull request closed without merging**